packages/backend/test/yeomanui.spec.ts

```diff
@@ -504,6 +504,29 @@ describe("yeomanui unit test", () => {
       });
     });
 
+    it("generator with 'hidden' category should not be shown'", async () => {
```

Comment:
please add more use cases for 'hidden' option - multiple categories allowed, no categories declared and etc.